packages/cap-config-writer/src/cap-config/index.ts

```diff
@@ -42,21 +44,24 @@ export async function enableCdsUi5Plugin(basePath: string, fs?: Editor): Promise
  *
  * @param basePath - root path of the CAP project, where package.json is located
  * @param [fs] - optional: the memfs editor instance
+ * @param {CdsVersionInfo} [cdsVersionInfo] - Optional. If provided will be used instead of parsing the package.json file to determine the cds version.
  * @returns true: cds-plugin-ui5 and all prerequisites are fulfilled; false: cds-plugin-ui5 is not enabled or not all prerequisites are fulfilled
  */
-export async function checkCdsUi5PluginEnabled(basePath: string, fs?: Editor): Promise<boolean>;
+export async function checkCdsUi5PluginEnabled(basePath: string, fs?: Editor, cdsVersionInfo?: CdsVersionInfo): Promise<boolean>;
```

Comment:
This is a breaking API change. The `moreInfo` param should be before the `cdsVersionInfo`

Reply:
@IainSAP so I have added `cdsVersionInfo` as the fourth argument. Let me know if its okay?

Reply:
Perfect, this way we still get proper type support on the return types based on the args passed.